remotedev/create.py

```diff
@@ -0,0 +1,180 @@
+# This script takes one argument: the name of the GitHub user for whom you want
+# to create a Zulip developer environment. Requires Python 3.
+#
+# Requires python-digitalocean library:
+# https://github.com/koalalorenzo/python-digitalocean
+#
+# Also requires Digital Ocean team membership for Zulip and api token:
+# https://cloud.digitalocean.com/settings/api/tokens
+#
+# Copy conf.ini-template to conf.ini and populate with your api token.
+#
+# usage: python3 create.py <username>
+
+import sys
+import configparser
+import urllib.request
+import json
+import digitalocean
+import time
+import argparse
+import os
+
+# initiation argument parser
+parser = argparse.ArgumentParser(description='Create a Zulip dev droplet based.')
+parser.add_argument("username", help="Github username for whom you want to create a Zulip dev droplet")
+
+def get_config():
+    config = configparser.ConfigParser()
+    config.read(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'conf.ini'))
+    return config
+
+def user_exists(username):
+    print("Checking to see if GitHub user {0} exists...".format(username))
+    apiurl_user = "https://api.github.com/users/{0}".format(username)
+    try:
+        response = urllib.request.urlopen(apiurl_user)
+        userdata = json.loads(response.read().decode())
+        print("...user exists!")
+        return True
+    except urllib.error.HTTPError as err:
+        print(err)
+        print("Does the github user {0} exist?".format(username))
+        sys.exit(1)
+
+def get_keys(username):
+    print("Checking to see that GitHub user has available public keys...")
+    apiurl_keys = "https://api.github.com/users/{0}/keys".format(username)
+    try:
+        response = urllib.request.urlopen(apiurl_keys)
+        userkeys = json.loads(response.read().decode())
+        print("...public keys found!")
+        return userkeys
+    except urllib.error.HTTPError as err:
+        print(err)
+        print("Has user {0} added ssh keys to their github account?".format(username))
+        sys.exit(1)
+
+def fork_exists(username):
+    print("Checking to see GitHub user has forked zulip/zulip...")
+    apiurl_fork = "https://api.github.com/repos/{0}/zulip".format(username)
+    try:
+        response = urllib.request.urlopen(apiurl_fork)
+        userfork = json.loads(response.read().decode())
+        print("...fork found!")
+        return True
+    except urllib.error.HTTPError as err:
+        print(err)
+        print("Has user {0} forked zulip/zulip?".format(username))
+        sys.exit(1)
+
+def droplet_exists(my_token, username):
```

Comment:
Since this method has side effects, I would call it something like `exit_early_if_droplet_exists`.